action/protocol/staking/vote_reviser.go

```diff
@@ -35,11 +37,57 @@ func (vr *VoteReviser) Revise(csm CandidateStateManager, height uint64) error {
 		if err != nil {
 			return err
 		}
+		if vr.needCorrectCands(height) {
+			if err := vr.correctAliasCands(csm, cands); err != nil {
+				return err
+			}
+			if csm, err = vr.reconstructCandCenter(csm); err != nil {
```

Comment:
the csm will be overwritten, but it won't be returned. so, the parameters set for https://github.com/iotexproject/iotex-core/pull/3688/commits/0614e984744ade435eb6616b976e951b1110a7e2#diff-4c37354767dc7764b93f5ed545905b6ee391187134ad8f29c09a7c71dfc51518R80 won't be applied to the code called out of this function.

Reply:
next time a `csm` is generated by `NewCandidateStateManager()`, it will use the base view, which contains the correct candidate center of the `newCSM` created here